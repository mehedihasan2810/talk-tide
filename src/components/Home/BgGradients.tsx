import React from 'react'

const BgGradients = () => {
  return (
    <div className="absolute -top-40 left-1/2 -z-10 flex -translate-x-1/2 gap-24 ">
    <div className="h-[900px] w-40 rotate-45 rounded-full bg-gradient-to-r from-cyan-300 to-blue-300 opacity-20  blur-[30px]"></div>
    <div className="h-[900px] w-40 rotate-45 rounded-full bg-gradient-to-r from-violet-300 to-fuchsia-300 opacity-20  blur-[30px]"></div>
    <div className="h-[900px] w-40 rotate-45 rounded-full bg-gradient-to-r from-sky-300 to-indigo-300 opacity-20  blur-[30px]"></div>
    <div className="h-[900px] w-40 rotate-45 rounded-full bg-gradient-to-r from-purple-300 to-pink-300 opacity-20  blur-[30px]"></div>
  </div>
  )
}

export default BgGradients