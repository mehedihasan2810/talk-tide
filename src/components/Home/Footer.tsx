import Link from 'next/link'
import React from 'react'

const Footer = () => {
  return (
    <footer className="fixed bottom-0 left-0 flex w-full flex-wrap items-center justify-center gap-4 border-t border-t-primary/20 p-4 text-primary/50 sm:gap-6">
    <Link className="hover:text-primary" href="#">
      Privacy Policy
    </Link>
    <Link className="hover:text-primary" href="#">
      Terms
    </Link>
    <Link className="hover:text-primary" href="#">
      Cookies Policy
    </Link>
    <Link className="hover:text-primary" href="#">
      {" "}
      Â© Talk Tide 2023
    </Link>
  </footer>
  )
}

export default Footer