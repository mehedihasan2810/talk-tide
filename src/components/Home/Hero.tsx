import Link from "next/link";
import React from "react";
import { buttonVariants } from "../ui/button";

const Hero = () => {
  return (
    <div className="space-y-4 px-1 text-center sm:space-y-12">
      <div className="relative text-3xl font-semibold text-primary-foreground">
        <svg
          className="absolute left-1/2 top-0 -z-10
         h-[120%] w-[130%] -translate-x-1/2"
          width="390"
          height="100"
          viewBox="0 0 390 100"
          fill="none"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M15.7313 23.391C109.247 17.6251 202.879 17.179 296.54 16.1553C297.212 16.148 331.206 14.1268 312.8 17.3748C288.651 21.6365 264.284 24.06 239.956 27.0495C165.808 36.1608 92.1668 51.1955 17.3573 51.1955C-61.7473 51.1955 175.566 51.1955 254.671 51.1955C276.866 51.1955 299.061 51.1955 321.255 51.1955C337.706 51.1955 354.07 50.2428 370.279 53.3093C385.217 56.1353 356.375 57.2669 353.288 57.6995C329.843 60.9835 306.503 64.7998 282.801 65.5855C259.326 66.3637 235.932 65.9662 212.476 67.618C173.892 70.3352 135.291 74.5097 96.7871 78.187C76.3725 80.1367 55.5268 79.0182 35.4871 83.4715C32.8341 84.061 40.8187 84.7132 43.5358 84.7722C63.5749 85.2079 83.6547 84.8535 103.698 84.8535C142.017 84.8535 180.336 84.8535 218.655 84.8535C262.079 84.8535 306.333 83.3902 349.385 83.3902"
            stroke="#008080"
            strokeWidth="30"
            strokeLinecap="round"
          />
        </svg>
        Talk Tide
      </div>
      <h1 className="text-4xl font-bold leading-normal text-primary sm:text-5xl sm:leading-none">
        <span className="relative isolate inline-block align-baseline">
          <svg
            className="absolute left-1/2 top-[70%] -z-10 w-full -translate-x-1/2"
            width="253"
            height="40"
            viewBox="0 0 253 40"
            fill="none"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M15.2446 24.7346C13.0311 20.7503 37.0529 17.4177 40.1223 17.4177C49.6272 17.4177 58.4559 15.9543 67.9268 15.9543C85.4875 15.9543 103.048 15.9543 120.609 15.9543C160.142 15.9543 198.154 20.3445 237.68 20.3445"
              stroke="#008080"
              strokeWidth="20"
              strokeLinecap="round"
            />
          </svg>

          <span className=""> Connect </span>
        </span>{" "}
        With Your Favorite Ones
      </h1>
      <Link
        href="/chat"
        className={buttonVariants({ className: "text-[18px] ", size: "lg" })}
      >
        Connect Now
      </Link>
    </div>
  );
};

export default Hero;
