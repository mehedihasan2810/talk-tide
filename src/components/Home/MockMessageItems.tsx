import React from "react";
import { Avatar, AvatarFallback, AvatarImage } from "../ui/avatar";

const MockMessageItems = () => {
  return (
    <div className="absolute inset-0 -z-[9] mx-auto max-w-[1920px]">
      {/* top left messages */}
      <div className="absolute left-[3%] top-[11%] flex items-end gap-1  lg:left-[8%] lg:top-[13%] xl:left-[15%]">
        <Avatar className="h-7 w-7">
          <AvatarImage
            className="object-cover"
            src="https://github.com/shadcn.png"
          />
          <AvatarFallback>M</AvatarFallback>
        </Avatar>

        <div className="flex flex-col items-start">
          <span className="ml-2 text-sm text-zinc-600">Morty</span>
          <div className="rounded-3xl rounded-bl-none bg-primary p-4 px-5 py-2 text-base text-primary-foreground shadow">
            Hola! 👋
          </div>
        </div>
      </div>

      <div className="absolute left-[10%] top-[18%] flex items-end gap-1 lg:left-[15%] lg:top-[20%] xl:left-[20%]">
        <div className="flex flex-col items-end">
          <span className="mr-2 text-sm text-zinc-600">Rick</span>
          <div className=" rounded-3xl rounded-br-none bg-primary-foreground p-4 px-5 py-2 text-base text-black shadow">
            Hello! 👍
          </div>
        </div>

        <Avatar className="h-7 w-7">
          <AvatarImage className="object-cover" src="/rick.png" />
          <AvatarFallback>M</AvatarFallback>
        </Avatar>
      </div>

      <div className="absolute hidden items-end gap-1 lg:left-[13%] lg:top-[28%]  lg:flex xl:left-[19%]">
        <div className="flex flex-col items-end">
          <span className="mr-2 text-sm text-zinc-600">Norty</span>
          <div className="rounded-3xl rounded-br-none bg-primary-foreground p-4 px-5 py-2 text-base text-black shadow">
            Bonjour! 👋
          </div>
        </div>

        <Avatar className="h-7 w-7">
          <AvatarImage className="object-cover" src="/morty.png" />
          <AvatarFallback>M</AvatarFallback>
        </Avatar>
      </div>

      {/* bottom left messages */}

      <div className="absolute hidden items-end gap-1 md:bottom-[21%] md:left-[3%] md:flex  xl:bottom-[26%] xl:left-[13%]">
        <Avatar className="h-7 w-7">
          <AvatarImage
            className="object-cover"
            src="https://images.pexels.com/photos/769772/pexels-photo-769772.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
          />
          <AvatarFallback>M</AvatarFallback>
        </Avatar>

        <div className="rounded-3xl rounded-bl-none bg-primary p-4 px-5 py-2 text-base text-primary-foreground shadow">
          Knock, knock. 😁
        </div>
      </div>

      <div className="absolute hidden items-end gap-1 md:bottom-[15%] md:left-[10%] md:flex xl:bottom-[20%] xl:left-[20%]">
        <div className="rounded-3xl rounded-br-none bg-primary-foreground p-4 px-5 py-2 text-base text-black shadow">
          Who&#39;s there? 😒
        </div>

        <Avatar className="h-7 w-7">
          <AvatarImage
            className="object-cover"
            src="https://images.pexels.com/photos/3228892/pexels-photo-3228892.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
          />
          <AvatarFallback>M</AvatarFallback>
        </Avatar>
      </div>

      {/* top right messages */}

      <div className="absolute  hidden items-end gap-1 md:right-[10%] md:top-[15%] md:flex  xl:right-[20%] xl:top-[25%]">
        <Avatar className="h-7 w-7">
          <AvatarImage className="object-cover" src="/joey.jpg" />
          <AvatarFallback>M</AvatarFallback>
        </Avatar>

        <div className="rounded-3xl rounded-bl-none bg-primary p-4 px-5 py-2 text-base text-primary-foreground shadow">
          How you doing? 😏
        </div>
      </div>

      <div className="absolute hidden items-end gap-1 md:right-[3%] md:top-[21%] md:flex  xl:right-[15%] xl:top-[32%]">
        <div className="rounded-3xl rounded-br-none bg-primary-foreground p-4 px-5 py-2 text-base text-black shadow">
          That&#39;s what she said 🤣
        </div>

        <Avatar className="h-7 w-7">
          <AvatarImage className="object-cover" src="/michael-scott.jpg" />
          <AvatarFallback>M</AvatarFallback>
        </Avatar>
      </div>

      {/* bottom right messages */}

      <div className="absolute bottom-[21%] right-[5%] flex items-end gap-1 xl:bottom-[21%] xl:right-[15%]">
        <Avatar className="h-7 w-7">
          <AvatarImage
            className="object-cover"
            src="https://images.pexels.com/photos/1427889/pexels-photo-1427889.jpeg?auto=compress&cs=tinysrgb&w=1600"
          />
          <AvatarFallback>M</AvatarFallback>
        </Avatar>

        <div className="rounded-3xl rounded-bl-none bg-primary p-4 px-5 py-2 text-base text-primary-foreground shadow">
          Take care of yourself mom❤ <br /> bye👋
        </div>
      </div>

      <div className="absolute bottom-[15%] right-[2%] flex items-end gap-1  lg:right-[3%] xl:right-[12%]">
        <div className="rounded-3xl rounded-br-none bg-primary-foreground p-4 px-5 py-2 text-base text-black shadow">
          You too son bye❤
        </div>

        <Avatar className="h-7 w-7">
          <AvatarImage
            className="object-cover"
            src="https://images.pexels.com/photos/1197132/pexels-photo-1197132.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
          />
          <AvatarFallback>M</AvatarFallback>
        </Avatar>
      </div>

      {/* ------------------------- */}
    </div>
  );
};

export default MockMessageItems;
