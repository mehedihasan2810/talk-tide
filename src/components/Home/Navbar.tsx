"use client";

import { ChevronRightIcon } from "@heroicons/react/20/solid";
import Link from "next/link";
import React from "react";
import { Button, buttonVariants } from "../ui/button";
import { cn } from "@/lib/utils";
import { signOut, useSession } from "next-auth/react";
import { useToast } from "../ui/use-toast";
const Navbar = () => {
  const { data: session } = useSession();
  const { toast } = useToast();

  return (
    <div className="fixed left-0 top-0 z-10  w-full border-b border-b-primary/40">
      <div className="mx-auto flex w-full max-w-7xl items-center justify-between px-3 py-2 sm:py-4">
        <div className="relative text-2xl font-semibold text-primary-foreground">
          <svg
            className="absolute left-1/2 top-0 -z-10
         h-[120%] w-[130%] -translate-x-1/2"
            width="390"
            height="100"
            viewBox="0 0 390 100"
            fill="none"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M15.7313 23.391C109.247 17.6251 202.879 17.179 296.54 16.1553C297.212 16.148 331.206 14.1268 312.8 17.3748C288.651 21.6365 264.284 24.06 239.956 27.0495C165.808 36.1608 92.1668 51.1955 17.3573 51.1955C-61.7473 51.1955 175.566 51.1955 254.671 51.1955C276.866 51.1955 299.061 51.1955 321.255 51.1955C337.706 51.1955 354.07 50.2428 370.279 53.3093C385.217 56.1353 356.375 57.2669 353.288 57.6995C329.843 60.9835 306.503 64.7998 282.801 65.5855C259.326 66.3637 235.932 65.9662 212.476 67.618C173.892 70.3352 135.291 74.5097 96.7871 78.187C76.3725 80.1367 55.5268 79.0182 35.4871 83.4715C32.8341 84.061 40.8187 84.7132 43.5358 84.7722C63.5749 85.2079 83.6547 84.8535 103.698 84.8535C142.017 84.8535 180.336 84.8535 218.655 84.8535C262.079 84.8535 306.333 83.3902 349.385 83.3902"
              stroke="#008080"
              strokeWidth="30"
              strokeLinecap="round"
            />
          </svg>
          Talk Tide
        </div>

        <nav className="relative inline-flex items-center gap-6">
          <Link
            className="inline-flex items-center text-primary hover:underline"
            href="/"
          >
            Home
          </Link>
          <Link
            className="inline-flex items-center text-primary hover:underline"
            href="/chat"
          >
            My Inbox
          </Link>
          <Link
            className="hidden items-center text-primary hover:underline sm:inline-flex"
            href="#"
          >
            Features <ChevronRightIcon className="h-6 w-6" />
          </Link>
          <Link
            className="hidden text-primary hover:underline sm:inline-block"
            href="#"
          >
            Help Center
          </Link>
          {session ? (
            <Button
              onClick={async () => {
                await signOut();
                toast({
                  description: "Successfully logged out",
                  variant: "success",
                });
              }}
              variant="outline"
            >
              Log out
            </Button>
          ) : (
            <Link
              href="/auth"
              className={cn(buttonVariants({ variant: "outline" }))}
            >
              Login
            </Link>
          )}
        </nav>
      </div>
    </div>
  );
};

export default Navbar;
