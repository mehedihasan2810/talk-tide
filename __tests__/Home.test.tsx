import Home from "@/app/(Home)/page";
import { render, screen } from "@testing-library/react";

describe("Home Component", () => {
  test("Should render properly", async () => {
    const { asFragment } = render(<Home />);

    const heading = screen.getByRole("heading", {
      level: 1,
      name: "Connect With Your Favorite Ones",
    });
    const btn = screen.getByRole("link", { name: "Connect Now" });
    const title = screen.getByText("Talk Tide");

    expect(heading).toBeInTheDocument();
    expect(btn).toBeInTheDocument();
    expect(title).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
