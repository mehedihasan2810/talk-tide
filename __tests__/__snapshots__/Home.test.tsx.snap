// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Should render properly 1`] = `
<DocumentFragment>
  <section
    class="relative isolate grid h-screen w-screen place-items-center overflow-hidden"
  >
    <div
      class="space-y-4 px-1 text-center sm:space-y-12"
    >
      <div
        class="relative text-3xl font-semibold text-primary-foreground"
      >
        <svg
          class="absolute left-1/2 top-0 -z-10   h-[120%] w-[130%] -translate-x-1/2"
          fill="none"
          height="100"
          viewBox="0 0 390 100"
          width="390"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M15.7313 23.391C109.247 17.6251 202.879 17.179 296.54 16.1553C297.212 16.148 331.206 14.1268 312.8 17.3748C288.651 21.6365 264.284 24.06 239.956 27.0495C165.808 36.1608 92.1668 51.1955 17.3573 51.1955C-61.7473 51.1955 175.566 51.1955 254.671 51.1955C276.866 51.1955 299.061 51.1955 321.255 51.1955C337.706 51.1955 354.07 50.2428 370.279 53.3093C385.217 56.1353 356.375 57.2669 353.288 57.6995C329.843 60.9835 306.503 64.7998 282.801 65.5855C259.326 66.3637 235.932 65.9662 212.476 67.618C173.892 70.3352 135.291 74.5097 96.7871 78.187C76.3725 80.1367 55.5268 79.0182 35.4871 83.4715C32.8341 84.061 40.8187 84.7132 43.5358 84.7722C63.5749 85.2079 83.6547 84.8535 103.698 84.8535C142.017 84.8535 180.336 84.8535 218.655 84.8535C262.079 84.8535 306.333 83.3902 349.385 83.3902"
            stroke="#008080"
            stroke-linecap="round"
            stroke-width="30"
          />
        </svg>
        Talk Tide
      </div>
      <h1
        class="text-4xl font-bold leading-normal text-primary sm:text-5xl sm:leading-none"
      >
        <span
          class="relative isolate inline-block align-baseline"
        >
          <svg
            class="absolute left-1/2 top-[70%] -z-10 w-full -translate-x-1/2"
            fill="none"
            height="40"
            viewBox="0 0 253 40"
            width="253"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M15.2446 24.7346C13.0311 20.7503 37.0529 17.4177 40.1223 17.4177C49.6272 17.4177 58.4559 15.9543 67.9268 15.9543C85.4875 15.9543 103.048 15.9543 120.609 15.9543C160.142 15.9543 198.154 20.3445 237.68 20.3445"
              stroke="#008080"
              stroke-linecap="round"
              stroke-width="20"
            />
          </svg>
          <span
            class=""
          >
             Connect 
          </span>
        </span>
         With Your Favorite Ones
      </h1>
      <a
        class="inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 bg-primary text-white hover:opacity-80 h-11 rounded-md px-8 text-[18px] "
        href="/chat"
      >
        Connect Now
      </a>
    </div>
    <div
      class="absolute -top-40 left-1/2 -z-10 flex -translate-x-1/2 gap-24 "
    >
      <div
        class="h-[900px] w-40 rotate-45 rounded-full bg-gradient-to-r from-cyan-300 to-blue-300 opacity-20  blur-[30px]"
      />
      <div
        class="h-[900px] w-40 rotate-45 rounded-full bg-gradient-to-r from-violet-300 to-fuchsia-300 opacity-20  blur-[30px]"
      />
      <div
        class="h-[900px] w-40 rotate-45 rounded-full bg-gradient-to-r from-sky-300 to-indigo-300 opacity-20  blur-[30px]"
      />
      <div
        class="h-[900px] w-40 rotate-45 rounded-full bg-gradient-to-r from-purple-300 to-pink-300 opacity-20  blur-[30px]"
      />
    </div>
    <div
      aria-hidden="true"
      class="absolute inset-0 -z-[9] mx-auto max-w-[1920px]"
    >
      <div
        class="absolute left-[3%] top-[11%] flex items-end gap-1  lg:left-[8%] lg:top-[13%] xl:left-[15%]"
      >
        <span
          class="relative flex shrink-0 overflow-hidden rounded-full h-7 w-7"
        >
          <span
            class="flex h-full w-full items-center justify-center rounded-full bg-zinc-200"
          >
            M
          </span>
        </span>
        <div
          class="flex flex-col items-start"
        >
          <span
            class="ml-2 text-sm text-zinc-600"
          >
            Morty
          </span>
          <div
            class="rounded-3xl rounded-bl-none bg-primary p-4 px-5 py-2 text-base text-primary-foreground shadow"
          >
            Hola! üëã
          </div>
        </div>
      </div>
      <div
        class="absolute left-[10%] top-[18%] mt-1 flex items-end gap-1 sm:mt-0 lg:left-[15%] lg:top-[20%] xl:left-[20%]"
      >
        <div
          class="flex flex-col items-end"
        >
          <span
            class="mr-2 text-sm text-zinc-600"
          >
            Rick
          </span>
          <div
            class=" rounded-3xl rounded-br-none bg-primary-foreground p-4 px-5 py-2 text-base text-black shadow"
          >
            Hello! üëç
          </div>
        </div>
        <span
          class="relative flex shrink-0 overflow-hidden rounded-full h-7 w-7"
        >
          <span
            class="flex h-full w-full items-center justify-center rounded-full bg-zinc-200"
          >
            M
          </span>
        </span>
      </div>
      <div
        class="absolute hidden items-end gap-1 lg:left-[13%] lg:top-[28%]  lg:flex xl:left-[19%]"
      >
        <div
          class="flex flex-col items-end"
        >
          <span
            class="mr-2 text-sm text-zinc-600"
          >
            Norty
          </span>
          <div
            class="rounded-3xl rounded-br-none bg-primary-foreground p-4 px-5 py-2 text-base text-black shadow"
          >
            Bonjour! üëã
          </div>
        </div>
        <span
          class="relative flex shrink-0 overflow-hidden rounded-full h-7 w-7"
        >
          <span
            class="flex h-full w-full items-center justify-center rounded-full bg-zinc-200"
          >
            M
          </span>
        </span>
      </div>
      <div
        class="absolute hidden items-end gap-1 md:bottom-[21%] md:left-[3%] md:flex  xl:bottom-[26%] xl:left-[13%]"
      >
        <span
          class="relative flex shrink-0 overflow-hidden rounded-full h-7 w-7"
        >
          <span
            class="flex h-full w-full items-center justify-center rounded-full bg-zinc-200"
          >
            M
          </span>
        </span>
        <div
          class="rounded-3xl rounded-bl-none bg-primary p-4 px-5 py-2 text-base text-primary-foreground shadow"
        >
          Knock, knock. üòÅ
        </div>
      </div>
      <div
        class="absolute hidden items-end gap-1 md:bottom-[15%] md:left-[10%] md:flex xl:bottom-[20%] xl:left-[20%]"
      >
        <div
          class="rounded-3xl rounded-br-none bg-primary-foreground p-4 px-5 py-2 text-base text-black shadow"
        >
          Who's there? üòí
        </div>
        <span
          class="relative flex shrink-0 overflow-hidden rounded-full h-7 w-7"
        >
          <span
            class="flex h-full w-full items-center justify-center rounded-full bg-zinc-200"
          >
            M
          </span>
        </span>
      </div>
      <div
        class="absolute  hidden items-end gap-1 md:right-[10%] md:top-[15%] md:flex  xl:right-[20%] xl:top-[25%]"
      >
        <span
          class="relative flex shrink-0 overflow-hidden rounded-full h-7 w-7"
        >
          <span
            class="flex h-full w-full items-center justify-center rounded-full bg-zinc-200"
          >
            M
          </span>
        </span>
        <div
          class="rounded-3xl rounded-bl-none bg-primary p-4 px-5 py-2 text-base text-primary-foreground shadow"
        >
          How you doing? üòè
        </div>
      </div>
      <div
        class="absolute hidden items-end gap-1 md:right-[3%] md:top-[21%] md:flex  xl:right-[15%] xl:top-[32%]"
      >
        <div
          class="rounded-3xl rounded-br-none bg-primary-foreground p-4 px-5 py-2 text-base text-black shadow"
        >
          That's what she said ü§£
        </div>
        <span
          class="relative flex shrink-0 overflow-hidden rounded-full h-7 w-7"
        >
          <span
            class="flex h-full w-full items-center justify-center rounded-full bg-zinc-200"
          >
            M
          </span>
        </span>
      </div>
      <div
        class="absolute bottom-[21%] right-[5%] flex items-end gap-1 xl:bottom-[21%] xl:right-[15%]"
      >
        <span
          class="relative flex shrink-0 overflow-hidden rounded-full h-7 w-7"
        >
          <span
            class="flex h-full w-full items-center justify-center rounded-full bg-zinc-200"
          >
            M
          </span>
        </span>
        <div
          class="rounded-3xl rounded-bl-none bg-primary p-4 px-5 py-2 text-base text-primary-foreground shadow"
        >
          Take care of yourself mom‚ù§ 
          <br />
           byeüëã
        </div>
      </div>
      <div
        class="absolute bottom-[15%] right-[2%] flex items-end gap-1  lg:right-[3%] xl:right-[12%]"
      >
        <div
          class="rounded-3xl rounded-br-none bg-primary-foreground p-4 px-5 py-2 text-base text-black shadow"
        >
          You too son bye‚ù§
        </div>
        <span
          class="relative flex shrink-0 overflow-hidden rounded-full h-7 w-7"
        >
          <span
            class="flex h-full w-full items-center justify-center rounded-full bg-zinc-200"
          >
            M
          </span>
        </span>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
